prototype.tf

```diff
@@ -0,0 +1,147 @@
+/* 
+ * [BEGIN] Test File Upload
+ * This is just adding temporary files for testing the SQL queries.
+ * These should be removed when the pipelines for adding these files are copmleted,
+ * and the SQL queries should point to those.
+ */
+
+resource "google_storage_bucket_object" "file_pdccr" {
+ name                     = "covid_19_pdccr.csv"
+ bucket                   = google_storage_bucket.gcs_data_ingestion_landing_bucket.name
+ source                   = "./testing_data_files/Provisional_COVID-19_Death_Counts_by_County_and_Race.csv"
+}
+
+resource "google_storage_bucket_object" "file_ucf" {
+ name                     = "Urgent_Care_Facilities.csv"
+ bucket                   = google_storage_bucket.gcs_data_ingestion_landing_bucket.name
+ source                   = "./testing_data_files/Urgent_Care_Facilities.csv"
+}
+
+/*
+ * [END] Test File Upload
+ */
+
+ /*
+  * [BEGIN] BigQuery Setup
+  */
+
+# Create a BigQuery dataset
+resource "google_bigquery_dataset" "bq_dataset" {
+  dataset_id              = var.bq_dataset_name
+  location                = "US"
+}
+
+/*
+ * [BEGIN] Table Creation
+ * Manually create tables for pdccr and ucf for testing. Much like the CSV files, these
+ * are just temporary loads until they are dynamically loaded in by the ingest pipelines.
+ */
+
+# PDCCR table
+resource "google_bigquery_table" "bq_table_pdccr" {
+  dataset_id = google_bigquery_dataset.bq_dataset.dataset_id
+  table_id   = "cdc_pdccr"
+}
+
+# PDCCR load
+resource "google_bigquery_job" "bq_job_load_pdccr" {
+  job_id     = "pdccr_load${formatdate("YYYYMMDDhhmmss",timestamp())}"
+
+  load {
+    source_uris = [
+      "gs://${var.gcs_landing_bucket}/covid_19_pdccr.csv",
+    ]
+
+    destination_table {
+      project_id = google_bigquery_table.bq_table_pdccr.project
+      dataset_id = google_bigquery_table.bq_table_pdccr.dataset_id
+      table_id   = google_bigquery_table.bq_table_pdccr.table_id
+    }
+
+    skip_leading_rows = 1
+
+    write_disposition = "WRITE_TRUNCATE"
+    autodetect = true
+  }
+}
+
+# UCF table
+resource "google_bigquery_table" "bq_table_ucf" {
+  dataset_id = google_bigquery_dataset.bq_dataset.dataset_id
+  table_id   = "ucf"
+}
+
+# UCF load
+resource "google_bigquery_job" "bq_job_load_ucf" {
+  job_id     = "ucf_load${formatdate("YYYYMMDDhhmmss",timestamp())}"
+
+  load {
+    source_uris = [
+      "gs://${var.gcs_landing_bucket}/Urgent_Care_Facilities.csv",
+    ]
+
+    destination_table {
+      project_id = google_bigquery_table.bq_table_ucf.project
+      dataset_id = google_bigquery_table.bq_table_ucf.dataset_id
+      table_id   = google_bigquery_table.bq_table_ucf.table_id
+    }
+
+    skip_leading_rows = 1
+
+    write_disposition = "WRITE_TRUNCATE"
+    autodetect = true
+  }
+}
+
+/*
+ * [END] Table Creation
+ */
+
+resource "google_bigquery_table" "bqt_pdccrucfjoin" {
+  dataset_id = google_bigquery_dataset.bq_dataset.dataset_id
+  table_id   = "pdccrucfjoin"
+
+  labels = {
+    "derived-table" ="yes"
```

Comment:
nit: missing space

Reply:
Oof yeah that annoys me too. I changed it locally in my code on my laptop, but I don't know what happens if I do a git push again on this branch while a pull request is open.